src/index.html

```diff
@@ -4,9 +4,417 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="moyo-logo">
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+             href="#Apple"
+             class="nav__link is-active"
+            >
+             Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#Samsung"
+             class="nav__link"
+            >
+             Samsung
+            </a>
```

Comment:
Attributes need to be moved to a new line only when there are 3 or more of them, in other cases it is not necessary to move them, it needs to be corrected everywhere
```suggestion
            <a href="#Samsung" class="nav__link">
              Samsung
            </a>
```

Reply:
done